tests/unit/test_collection.py

```diff
@@ -57,9 +57,7 @@ def test_join_collection_no_sync_called():
         product_info=ProductInfo.for_testing(WorkspaceConfig),
     )
     with pytest.raises(ValueError):
-        account_installer.join_collection(
-            123,
-        )
+        account_installer.join_collection(123)
```

Comment:
@JCZuurmond Thanks for fixing the issue btw. This test is validate if the workspaces.json file is up to to date. If the mapping is not available it will throw a value error we log a message asking user to run sync-workspace-info cli cmd to sync the workspaces.json up to date and try again. collection_workspace can be null in which case the code will list all the available workspaces to join.